benchmarks/Bio/BCR.jmd

```diff
@@ -14,138 +14,322 @@ Catalyst network model to ODEs.
 
 
 ```julia
-using DiffEqBase, OrdinaryDiffEq, Catalyst, ReactionNetworkImporters,
-      Sundials, Plots, DiffEqDevTools, ODEInterface, ODEInterfaceDiffEq,
-      LSODA, TimerOutputs, LinearAlgebra, ModelingToolkit, BenchmarkTools
+if false

```

Comment:
what? That disables it?

Reply:
Basically, tying to figure out what went wrong, I disabled all code in the BCR and fceri models, trying to work myself down to figuring out what the problem is. The remaining models all run relatively fast, so shouldn't be a run time issue.